src/pages/VideoPlayer/VideoPlayer.page.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { ThemeContext } from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import {
+  PageContent,
+  VideoList,
+  VideoContent,
+  VideoInformationContainer,
+} from './VideoPlayer.styles';
+import { useAppDataContext } from '../../providers/AppData';
+import VideoCardSmall from '../../components/VideoCardSmall';
+import { useAuth } from '../../providers/Auth';
+import actions from '../../state/actions';
+import { useSingleVideo } from '../../utils/hooks/useSingleVideo';
+import { useRelatedVideos } from '../../utils/hooks/useRelatedVideos';
+
+const VideoPlayer = () => {
+  const { state, dispatch } = useAppDataContext();
+  const { authenticated } = useAuth();
+  const [isFavorite, setIsFavorite] = useState();
+
+  const { idVideo } = useParams();
+
+  useEffect(() => {
+    setIsFavorite(state.favorites.some((video) => video.id === idVideo));
+  }, [state.favorites, idVideo]);
+
+  useRelatedVideos(idVideo);
+  useSingleVideo(idVideo);
+
+  const { theme } = useContext(ThemeContext);
+
+  const toggleFavorite = () => {
+    dispatch({ type: actions.TOGGLE_FAVORITE, payload: idVideo });
+  };
+
+  const { videos, currentVideo } = state;
+
+  return (
+    <PageContent>
+      <VideoContent>
+        <iframe
+          title="youtube-player"
+          id="player"
+          type="text/html"
+          width="100%"
+          height="100%"
+          src={`http://www.youtube.com/embed/${idVideo}?enablejsapi=1`}
```

Comment:
Please use `https://` protocol, otherwise it won't work.

![Screen Shot 2020-12-09 at 16 13 23](https://user-images.githubusercontent.com/3632628/101696515-d53da300-3a3b-11eb-8d73-f0a0ca9ababe.png)